modules/ROOT/pages/release-notes.adoc

```diff
@@ -235,6 +235,35 @@ Use BPMN 2 importer for model exchange with third-party editors.
 
 == Bug fixes
 
+=== Fixes in Bonita 2021.1-0702 (2021-07-02)
+
+==== Fixes in Bonita Development Suite (Studio and UI Designer)
+
+* UID-453	- ResponsivenessHelp in editor help is not displayed
+* UID-369	- Property type changes of target widgets are not mapped when using Switch feature
+* UID-165	- When modifying & save an asset, if you press SUPR, the selected widget is removed
+
+
+==== Fixes in Bonita Runtime (including Portal)
+
+* RUNTIME-326	- Docker logs should not contain sensitive informations
```

Comment:
For all customers using previous versions of Bonita, we clearly expose that their logs contain sensitive informations. Is that critical? Should we check with Saghar?

Reply:
```suggestion
* RUNTIME-326	- Reduce information in Docker start logs 
```